scene/3d/skeleton_3d.h

```diff
@@ -101,8 +131,9 @@ class Skeleton3D : public Node3D {
 
 	bool animate_physical_bones = true;
 	Vector<Bone> bones;
```

Comment:
For another PR for sure, but in future we should convert this to a `LocalVector` instead. This array is manipulated much more than it's passed around. We will avoid a bunch of useless reallocations by using `LocalVector`.

Reply:
Since I was already updating the others, I changed `bones` to use a LocalVector as well. It doesn't seem to have changed anything functionally and only needed a few minor modifications to the source code of the Skeleton.